test/unit/spec/EnrollChoices_spec.js

```diff
@@ -78,24 +81,34 @@ function (Okta, OktaAuth, Util, EnrollChoicesForm, Beacon, Expect, FactorUtil, R
       factorType: 'assertion:oidc'
     }
   };
+  var enrolledFactor = {
```

Comment:
use const so we dont accidentally override in our large test files.

Reply:
👍